common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExternalShuffleBlockResolver.java

```diff
@@ -183,12 +190,7 @@ public ManagedBuffer getBlockData(String appId, String execId, String blockId) {
         String.format("Executor is not registered (appId=%s, execId=%s)", appId, execId));
     }
 
-    if ("sort".equals(executor.shuffleManager) || "tungsten-sort".equals(executor.shuffleManager)) {
-      return getSortBasedShuffleBlockData(executor, shuffleId, mapId, reduceId);
-    } else {
-      throw new UnsupportedOperationException(
-        "Unsupported shuffle manager: " + executor.shuffleManager);
-    }
+    return getSortBasedShuffleBlockData(executor, shuffleId, mapId, reduceId);
```

Comment:
I think it is not necessary. Because if it is other shuffleManager, it can not find it from executor list. As before, it will throw a UnsupportedOperationException. see: [https://github.com/apache/spark/blob/5e92583d38e11d39deb429a39725443111205a4a/common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExternalShuffleBlockResolver.java#L181](url)